internals/core_models.py

```diff
@@ -954,3 +954,206 @@ def put(self, notify=True, **kwargs):
   anticipated_spec_changes = ndb.StringProperty()
 
   finch_url = ndb.StringProperty()
+
+
+# Note: This class is not used yet.
+class FeatureEntry(ndb.Model):  # Copy from Feature
+  """This is the main representation of a feature that we are tracking."""
+
+  # Metadata: Creation and updates.
+  created = ndb.DateTimeProperty(auto_now_add=True)
+  updated = ndb.DateTimeProperty()
+  accurate_as_of = ndb.DateTimeProperty()
+  creator = ndb.StringProperty()
+  updater = ndb.StringProperty()
+
+  # Metadata: Access controls
+  owners = ndb.StringProperty(repeated=True)  # copy from owner
+  editors = ndb.StringProperty(repeated=True)
+  unlisted = ndb.BooleanProperty(default=False)
+  deleted = ndb.BooleanProperty(default=False)
+
+  # Descriptive info.
+  name = ndb.StringProperty(required=True)
+  summary = ndb.TextProperty(required=True)
+  category = ndb.IntegerProperty(required=True)
+  blink_components = ndb.StringProperty(repeated=True)
+  star_count = ndb.IntegerProperty(default=0)
+  search_tags = ndb.StringProperty(repeated=True)
+  feature_notes = ndb.TextProperty()  # copy from comments
+
+  # Metadata: Process information
+  feature_type = ndb.IntegerProperty(default=FEATURE_TYPE_INCUBATE_ID)
+  intent_stage = ndb.IntegerProperty(default=0)
+  bug_url = ndb.StringProperty()
+  launch_bug_url = ndb.StringProperty()
+
+  # Implementation in Chrome
+  impl_status_chrome = ndb.IntegerProperty(required=True)
+  flag_name = ndb.StringProperty()
+  ongoing_constraints = ndb.TextProperty()
+
+  # Gate: Adoption
+  motivation = ndb.TextProperty()
```

Comment:
What's your implementation timeline? I'm still trying to work out whether this should be merged wit the summary or renamed in some way.

Reply:
Ping!

Reply:
We might get to this part of the data migration in the next 3 or 4 weeks.  We can always make changes after that, but it would be better to make changes before if possible.